client/js/components/user/CustomerSettings/CustomerSettingsBranding.vue

```diff
@@ -47,12 +48,12 @@
           style="max-width: 300px;">
         <dp-checkbox
           id="r_logoDelete"
+          v-model="logoDelete"
           :label="{
             bold: true,
             text: Translator.trans('logo.delete')
           }"
-          name="r_logoDelete"
-          value-to-send="deleteLogo" />
```

Comment:
needs to be adjusted after renaming, too

Reply:
value-to-send is unnecessary after the refactoring. This was used for the post request handling

Reply:
this should be removed then in that case i think

Reply:
someone removed this already in another commit